### PR TITLE
Added B_tree in C

### PR DESCRIPTION
**Pull request by benda1997**

### CODE REVIEW

File: code/data_structures/B_Tree/b_tree.c [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+void inorder (BTREE root);
+void postorder (BTREE root);
+�
+/**********************
+�* binarytree.c:
+�***********************/
+#include <assert.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "binarytree.h"
```
Comment (AdiChat) [inline code/data_structures/B_Tree/b_tree.c]: `binarytree.h` file is not present in this commit. Kindly add it 👍

File: code/data_structures/B_Tree/b_tree.c [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+{
+    if (root != NULL) {
+        postorder(root -> left);
+        postorder(root -> right);
+        printf("%c ", root->d);
+    }
+}
+�
+/***************************
+�* pgm.c
```
Comment (AdiChat) [inline code/data_structures/B_Tree/b_tree.c]: If this is intended to be a different file, kind add it like that 👍

File: code/data_structures/B_Tree/b_tree.c [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+�***************************/
+#include <assert.h>
+#include <stdio.h>
+#include <stdlib.h>
+�
+#include "binarytree.c"
```
Comment (AdiChat) [inline code/data_structures/B_Tree/b_tree.c]: `binarytree.c` is missing from this commit. Kindly add it 👍

File: code/data_structures/B_Tree/b_tree.c [hunk 4]
```diff
@@ -0,0 +1,121 @@
+/*
```
Comment (AdiChat) [inline code/data_structures/B_Tree/b_tree.c]: This is a binary tree not a B Tree

**Comment by AdiChat** [comment]
B Tree is different from Binary tree. See B Tree on [Wikipedia](https://en.wikipedia.org/wiki/B-tree)

Binary tree in `C` is also required but in your code, `binarytree.h` and `binarytree.c` are missing which should be included 👍 

Kindly add the above files to get this merged in `binary tree` folder.

**Comment by AdiChat** [comment]
B tree in `C++` already exists 👍

**Comment by arnavb** [comment]
@AdiChat Please close!

**Comment by AdiChat** [comment]
@benda1997 We have closed this pull request but you can always open another pull request for an addition you wish to make. :+1:

**Pull request closed without merging**